### PR TITLE
Updated readme with full usage

### PR DESCRIPTION
**Pull request by pierresouchay**

### CODE REVIEW

File: haproxy/haproxy_cmd/run.go [hunk 1]
```diff
@@ -69,6 +69,7 @@ func Start(sd *lib.Shutdown, cfg Config) (*dataplane.Dataplane, error) {
 
 		err = dataplaneClient.Ping()
 		if err != nil {
+			fmt.Println("*****\n* SOUCHAY: wait for dataplane to be up\n*****")
```
Comment (aiharos) [inline haproxy/haproxy_cmd/run.go:72]: A forgotten debug Println?
```suggestion
```

File: consul/logger.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+type Logger interface {
+	// Debugf Display debug message
+	Debugf(format string, args ...interface{})
+	// Infof Display info message
+	Infof(format string, args ...interface{})
+	// Warnf Display warning message
+	Warnf(format string, args ...interface{})
+	// Errorf Display error message
+	Errorf(format string, args ...interface{})
+}
```
Comment (aiharos) [inline consul/logger.go:13]: Missing newline
```suggestion
}
```

File: README.md [hunk 3]
```diff
@@ -23,16 +23,51 @@ To handle intentions, HAProxy Connect, sets up a SPOE filter on the application
...
+  -sidecar-for string
+    	The consul service id to proxy
+  -sidecar-for-tag string
+    	The consul service id to proxy
+  -stats-addr string
+    	Listen addr for stats server
+  -stats-service-register
+    	Register a consul service for connect stats
+  -token string
+    	Consul ACL token
```
Comment (aiharos) [inline README.md:72]: This is a copy-paste duplicate of the lines above?
```suggestion
```

Comment (pierresouchay) [reply README.md:72]: I got lost in my branches... fixed in next commit

Comment (aiharos) [reply README.md:72]: Not a problem! I will merge as soon as it is fixed!

Comment (aiharos) [reply README.md:72]: Actually I can commit and merge

**Comment by pierresouchay** [comment]
@aiharos All clean now

**Comment by aiharos** [comment]
> @aiharos All clean now

Yup, looks good, merging.

**Pull request merged**